### ISSUE TITLE
R assignment 3: Statistics-2-R import of data

### ISSUE THREAD

**Issue by andrewmarx** [datacarpentry/semester-biology#232]
The last problem in the assignment requires students to import a data file that they supposedly created, but nowhere else in the assignment has them export a data file.

**Comment by ethanwhite** [comment]
Thanks. I'm working on this assignment now and expect to be pulling that question since we haven't gotten to statistics yet. @brymz - was there a reason that question was included here that you remember?

**Comment by brymz** [comment]
The original plan for exercises in the `dplyr` assignment was 'Modules',
'Scientific 0', 'Combining Basics', and 'Statistics 2'. They build on an
analysis of the `shrub_volume_experiment.csv` file.

With moving the `dplyr` assignment before the 'basic programming' section,
I had to pull 'Combining Basics' because it calls for writing a function
and using a `for` loop. I left `Statistics 2` because the problem is just a
simple function call of `aov()`, but I failed to notice that I pulled the
source of data with `Combining Basics`. The problem can be solved with a
simple data.frame of `experiment` and `volume`.

I will be putting in 6 hrs of work tomorrow and can prioritize a solution.
Happy to take direction from you folks on the ground. I worry that the
assignment becomes pretty sparse if it's just 'Modules' and 'Scientific 0'.
